ChaosMod/Effects/db/PedsNapTime.cpp

```diff
@@ -0,0 +1,37 @@
+#include <stdafx.h>
+
+/*
+	Effect by Eli Rickard
+*/
+
+static void OnStop()
+{
+	REMOVE_ANIM_DICT("amb@world_human_bum_slumped@male@laying_on_left_side@idle_a");
+}
+
+static void OnTick()
+{
+	SET_CLOCK_TIME(21, 30, 50);
+
+	for (auto ped : GetAllPeds())
+	{
+		if (IS_PED_IN_ANY_VEHICLE(ped, true))
+		{
+			Vehicle pedVeh = GET_VEHICLE_PED_IS_IN(ped, true);
+			TASK_LEAVE_VEHICLE(ped, pedVeh, 256);
+			BRING_VEHICLE_TO_HALT(pedVeh, 0.1f, 10, 0);
+		}
+	}
```

Comment:
Thinking about it for 2 seconds, this could be done in the `OnStart`, unless you want any ped that is spawned in _any_ vehicle to do this? I.e., if you drive a bit and new vehicles spawn

Reply:
I wanted everyone new that spawned it after the fact, to also be sleeping given that it is a timed effect but i'll fix the Getting of all peds twice, now.